.travis.yml

```diff
@@ -2,22 +2,18 @@ sudo: false
 language: node_js
 node_js:
   - '6'
-  - '5'
-  - '4'
```

Comment:
This may be solved by running them serially: https://docs.travis-ci.com/user/customizing-the-build#Limiting-Concurrent-Builds